dev-stack/stack.docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3.1"
```

Comment:
this addition was not on purpose, should this file be removed? idk if its in main

Reply:
Yes I think it should be removed, this file is now named "example.docker-compose.yml"